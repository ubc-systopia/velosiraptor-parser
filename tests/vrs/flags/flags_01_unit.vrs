segment Foo {
    flags = { foo, bar };
    flags = { FOO, BAR };

    // with trailing comma
    flags = { foo, bar, };
    flags = { FOO, BAR, };

    // empty flags
    flags = { };

    // repetition
    flags = { FOO, FOO, };
}