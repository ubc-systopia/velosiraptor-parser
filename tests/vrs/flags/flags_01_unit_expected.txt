VelosiParseTree(tests/vrs/flags/flags_01_unit.vrs)
---------------------------------------------
segment Foo {
  flags = { foo, bar };
  flags = { FOO, BAR };
  flags = { foo, bar };
  flags = { FOO, BAR };
  flags = {  };
  flags = { FOO, FOO };
}


---------------------------------------------
